Update ms-cloud-germany-transition-phases.md (#4447)
MicrosoftDocs/microsoft-365-docs: microsoft-365/enterprise/ms-cloud-germany-transition-phases.md

In case you have set a DNS CNAME called _msoid_ in one or many DNS namespaces that you own, you have to remove the CNAME until the end of phase 8 at the latest. You can remove the CNAME _msoid_ any time before the end of phase 8. See the [prework for DNS](ms-cloud-germany-transition-add-pre-work.md#DNS).

In case you are using single sign on for Office 365 and Azure in the Microsoft Cloud Deutschland instance, you must prepare and schedule your Azure subscription migration accordingly. Make sure that you understand the [prework for Microsoft Azure](ms-cloud-germany-transition-add-pre-work.md#Microsoft-Azure).

## Before Phase 2 starts

If you are using ADFS, make sure to [back up your ADFS configuration before and after adding the relying party trust](ms-cloud-germany-transition-azure-ad.md) for the Office 365 Global service.

## Subscription transfer (Phase 3)

**Applies to**: All customers with an Office 365 tenant hosted in the Microsoft Cloud Deutschland (MCD)

| Step(s) | Description | Impact |
|:-------|:-------|:-------|

> | Subscriptions are transferred, and licenses are reassigned. | Corresponding Office 365 Global services subscriptions are purchased in the Office 365 Global instance for the transferred Microsoft Cloud Deutschland subscriptions. Users with assigned Microsoft Cloud Deutschland licenses will be assigned licenses in the Office 365 Global instance. Legacy Microsoft Cloud Deutschland subscriptions are removed from the Office 365 services tenant on completion.| <ul><li>Changes to existing subscriptions will be blocked (for example, no new subscription purchases or seat count changes) during this phase.</li><li>License assignment changes will be blocked.</li><li>The Microsoft Cloud Deutschland subscription will be migrated to corresponding Office 365 Global services subscription. The Office 365 Global services offer of that subscription is defined by Microsoft (also known as _Offer mapping_).</li><li>The number of features (service plans) offered by Office 365 services can be larger than in the original Microsoft Cloud Deutschland offer. User licenses in Office 365 services will be equivalently assigned to similar Microsoft Cloud Deutschland features (service plans). User licenses of all users will be automatically assigned to the new features. The admin needs to take an explicit action to disable those licenses, if needed. </li><li>When subscription migration is complete, both Office 365 services and Microsoft Cloud Deutschland  subscriptions will be visible in the Office 365 Admin Portal, with the status of Microsoft Cloud Deutschland subscriptions as _deprovisioned_. </li><li>Users will be reassigned licenses that are tied to the new Office 365 services subscriptions. Any customer processes that have dependencies on MCD subscriptions or SKU GUIDs will be broken and need to be revised with the Office 365 services offering. </li><li>New subscriptions in the Office 365 services will be purchased with the new term (monthly/quarterly/yearly), and the customer will receive a prorated refund for the unused balance of the Microsoft Cloud Deutschland subscription. </li><li>Partner Microsoft Cloud Deutschland tenants won't be migrated. CSP customers will be migrated to Office 365 services under the new Office 365 services tenant of the same partner. After customer migration, the partner can manage this customer only from the Office 365 services tenant. </li><li>Additional functionality is available (for example, Microsoft Planner and Microsoft Flow), unless disabled by tenant admin. For information about how to disable service plans that are assigned to users' licenses, see [Disable access to Microsoft 365 services while assigning user licenses](disable-access-to-services-while-assigning-user-licenses.md).</li></ul> |

Reviewer: ```suggestion
| Subscriptions are transferred, and licenses are reassigned. | Corresponding Office 365 Global services subscriptions are purchased in the Office 365 Global instance for the transferred Microsoft Cloud Deutschland subscriptions. Users with assigned Microsoft Cloud Deutschland licenses will be assigned licenses in the Office 365 Global instance. Legacy Microsoft Cloud Deutschland subscriptions are removed from the Office 365 services tenant on completion.| <ul><li>Changes to existing subscriptions will be blocked (for example, no new subscription purchases or seat count changes) during this phase.</li><li>License assignment changes will be blocked.</li><li>The Microsoft Cloud Deutschland subscription will be migrated to corresponding Office 365 Global services subscription. The Office 365 Global services offer of that subscription is defined by Microsoft (also known as _Offer mapping_).</li><li>The number of features (service plans) offered by Office 365 services can be larger than in the original Microsoft Cloud Deutschland offer. User licenses in Office 365 services will be equivalently assigned to similar Microsoft Cloud Deutschland features (service plans). User licenses of all users will be automatically assigned to the new features. The admin needs to take an explicit action to disable those licenses, if needed. </li><li>When subscription migration is complete, both Office 365 services and Microsoft Cloud Deutschland subscriptions will be visible in the Office 365 Admin Portal, with the status of Microsoft Cloud Deutschland subscriptions as _deprovisioned_. </li><li>Users will be reassigned licenses that are tied to the new Office 365 services subscriptions. Any customer processes that have dependencies on Microsoft Cloud Deutschland subscriptions or SKU GUIDs will be broken and need to be revised with the Office 365 services offering. </li><li>New subscriptions in the Office 365 services will be purchased with the new term (monthly/quarterly/yearly), and the customer will receive a prorated refund for the unused balance of the Microsoft Cloud Deutschland subscription. </li><li>Partner Microsoft Cloud Deutschland tenants won't be migrated. CSP customers will be migrated to Office 365 services under the new Office 365 services tenant of the same partner. After customer migration, the partner can manage this customer only from the Office 365 services tenant. </li><li>Additional functionality is available (for example, Microsoft Planner and Microsoft Flow), unless disabled by tenant admin. For information about how to disable service plans that are assigned to users' licenses, see [Disable access to Microsoft 365 services while assigning user licenses](disable-access-to-services-while-assigning-user-licenses.md).</li></ul> |
```

Reviewer: Remove the extra space between "Deutschland" and "subscriptions" in line 20.
Change "MCD" in line 22 to "Microsoft Cloud Deutschland".

||||

## SharePoint Online (Phase 4)

**Applies to**: All customers using SharePoint Online

> In case you are still using SharePoint 2013 workflows, limit SharePoint 2013 workflows the use during the SharePoint Online migration.

Reviewer: ```suggestion
In case you are still using SharePoint 2013 workflows, limit the use of SharePoint 2013 workflows during the SharePoint Online migration.
```

- Microsoft Cloud Deutschland customers whose SharePoint Online instance is migrated must update SharePoint Online PowerShell module/Microsoft.SharePointOnline.CSOM to version 16.0.20717.12000 or above. Otherwise, connections to SharePoint Online via PowerShell or the client-side object model will fail.

> [!NOTE]

>     > In case you are using eDiscovery, make sure you are aware of the [eDiscovery migration experience](ms-cloud-germany-transition-add-experience.md#eDiscovery-phase-4-to-the-end-of-phase-9)

Reviewer: ```suggestion
> In case you are using eDiscovery, make sure you are aware of the [eDiscovery migration experience](ms-cloud-germany-transition-add-experience.md#eDiscovery-phase-4-to-the-end-of-phase-9).
```

## Exchange Online (Phase 5)

**Applies to:** All customers using Exchange Online

> If you're using Exchange Online hybrid: Exchange Online Hybrid administrators  **must execute the Hybrid Configuration wizard (HCW) multiple times** as part of this transition. Apply the [Exchange prework](ms-cloud-germany-transition-add-pre-work.md#Exchange-Online-Hybrid-configuration), **before the migration step phase 5 begins,** Exchange Online hybrid customers must run the latest version of the Exchange Hybrid Configuration Wizard (HCW) in "Office 365 Germany" mode to prepare the on-premises configuration for the migration to Office 365 global services.

Reviewer: ```suggestion
If you're using Exchange Online hybrid: Exchange Online Hybrid administrators  **must execute the Hybrid Configuration wizard (HCW) multiple times** as part of this transition. Apply the [Exchange prework](ms-cloud-germany-transition-add-pre-work.md#Exchange-Online-Hybrid-configuration) **before the migration step phase 5 begins**. Exchange Online hybrid customers must run the latest version of the Exchange Hybrid Configuration Wizard (HCW) in "Office 365 Germany" mode to prepare the on-premises configuration for the migration to Office 365 global services.
```

Upon **completion of the migration phase 9** (when the Message Center notice is published), you need to run the HCW again using Office 365 Worldwide settings to point your on-premises systems to the Office 365 Global services.

If you want to modify user photos during phase 5, see [Set-UserPhoto](ms-cloud-germany-transition-add-experience.md#Exchange-Online-Set-UserPhoto)

| Step(s) | Description | Impact |
|:-------|:-------|:-------|

> |Stop or delete any onboarding or offboarding mailbox moves, namely don't move mailboxes between Exchange on premises and Exchange Online  | This ensures the mailbox move requests don't fail with an error. | Failure to do so may result in failure of the service or Office clients. |

Reviewer: ```suggestion
|Stop or delete any onboarding or offboarding mailbox moves, namely don't move mailboxes between Exchange on-premises and Exchange Online.  | This ensures the mailbox move requests don't fail with an error. | Failure to do so may result in failure of the service or Office clients. |
```

## Skype for Business Online (Phase 7)

**Applies to:** All customers using Skype for Business Online

> Make sure that you are familiar with the [prework for your Skype for Business Online migration](ms-cloud-germany-transition-add-pre-work.md#Skype-for-Business-Online)

Reviewer: ```suggestion
Make sure that you are familiar with the [prework for your Skype for Business Online migration](ms-cloud-germany-transition-add-pre-work.md#Skype-for-Business-Online) procedure.
```

||||

> If you have to connect to Skype for Business Online with PowerShell after migration phase 9 has been completed use the follwoing  code to connect:

Reviewer: ```suggestion
If you have to connect to Skype for Business Online with PowerShell after migration phase 9 has been completed, use the following code to connect:
```

```powershell
Import-Module MicrosoftTeams
$userCredential = Get-Credential
Connect-MicrosoftTeams -Credential $userCredential -OverridePowershellUri "https://admin4E.online.lync.com/OcsPowershellOAuth"
```

## Dynamics 365 (Phase 8)

**Applies to:** All customers using Microsoft Dynamics 365

> Make sure that you are familiar with the [prework for your Microsoft Dynamics 365 installation](ms-cloud-germany-transition-add-pre-work.md#Dynamics-365)

Reviewer: ```suggestion
Make sure that you are familiar with the [prework for your Microsoft Dynamics 365 installation](ms-cloud-germany-transition-add-pre-work.md#Dynamics-365) procedure.
```

Office 365 tenants transitioning to the region "Germany" require all users to close, sign out from Office 365 and back in for all Office desktop applications (Word, Excel, PowerPoint, Outlook, etc.) and OneDrive for Business client after the tenant migration has reached phase 9. Signing out and in, allows the Office services to obtain new authentication tokens from the global Azure AD service.

> Make sure, you have done the [prework for mobile devices](ms-cloud-germany-transition-add-pre-work.md#Mobile)

Reviewer: ```suggestion
Make sure you have completed the [prework for mobile devices](ms-cloud-germany-transition-add-pre-work.md#Mobile) procedure.
```

| Step(s) | Description | Impact |
|:-------|:-------|:-------|
| Clients, Office Online during Office client cutover, Azure AD finalizes the tenant scope to point to the Office 365 services. | This configuration change enables Office clients to update and point to the Office 365 services endpoints. | <ul><li>Notify users to close _all_ Office apps and then sign back in (or force clients to restart and users to sign in) to enable Office clients to pick up the change. </li><li>Notify users and help desk staff that users *may* see an Office banner that prompts them to reactivate Office apps within 72 hours of the cutover. </li><li>All Office applications on personal machines must be closed, and users must sign out then sign in again. In the Yellow activation bar, sign in to reactivate against Office 365 services.</li><li>Shared machines will require actions that are similar to personal machines, and won't require a special procedure. </li><li>On mobile devices, users must sign out of apps, close them, and then sign in again. </li></ul>|
||||

## Line-of-business apps

> In case you have line-of-business apps, make sure you have done the [prework for line-of-business apps](ms-cloud-germany-transition-add-pre-work.md#line-of-business-apps)

Reviewer: ```suggestion
In case you have line-of-business apps, make sure you have completed the [prework for line-of-business apps](ms-cloud-germany-transition-add-pre-work.md#line-of-business-apps) procedure.
```

## Office Services

The most recently used (MRU) service in Office is a cutover from the Microsoft Cloud Deutschland to Office 365 Global services, not a migration. Only MRU links from the Office 365 Global services side will be visible after migration from the Office.com portal. MRU links from the Microsoft Cloud Deutschland aren't visible as MRU links in Office 365 Global services. In Office 365 Global services, MRU links are accessible only after the tenant migration has reached phase 9.

## Post migration

> Make sure you read the [post migration activities](ms-cloud-germany-transition-add-experience.md#Post-migration) and execute them accordingly.

Reviewer: ```suggestion
Make sure you read the [post migration activities](ms-cloud-germany-transition-add-experience.md#Post-migration) and execute them accordingly article.
```

> Make sure you read the [post migration activities](ms-cloud-germany-transition-add-experience.md#Post-migration) and execute them accordingly article.

Reviewer: ```suggestion
Make sure you read the [post migration activities](ms-cloud-germany-transition-add-experience.md#Post-migration) article and execute them accordingly.
```
